src/styles.css

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+body{
+  background-color: rgb(245, 245, 245);
```

Comment:
You have the same css rule on the top with *. I think one is overlapping the other. Make sure if both are needed.